extensions/blocks/map/map.php

```diff
@@ -40,6 +40,20 @@ function jetpack_get_mapbox_api_key() {
 	return Jetpack_Options::get_option( 'mapbox_api_key' );
 }
 
+/**
+ * Record a Tracks event every time the Map block is loaded on WordPress.com and Atomic.
+ */
+function jetpack_record_mapbox_wpcom_load_event() {
+	$event_name = 'wpcom_map_block_mapbox_load';
```

Comment:
I believe prefixing the event with `wpcom` implies the event is for Simple Sites (on wpcom). I don't think we need to track `wpcom` and `jetpack` events separately, it's probably more convenient to keep this as a single event.

What about simply `map_block_mapbox_load`?

Reply:
Or maybe `map_block_mapbox_wpcom_key_load`?

Reply:
Yeah I was on the fence about this event name. 🤔 
I guess `map_block_mapbox_wpcom_key_load` is the most descriptive and obvious.

> I don't think we need to track wpcom and jetpack events separately

Agreed. I don't see much value in knowing the site type for this event.